temporalcli/commands_test.go

```diff
@@ -317,15 +293,9 @@ func StartDevServer(t *testing.T, options DevServerOptions) *DevServer {
 		require.NoError(t, err)
 	}
 
-	success = true
 	return d
 }
 
-func (d *DevServer) Stop() {
```

Comment:
If I choose to invoke `StartDevServer` I should be able to choose when to stop it. I don't think we should assume lifetimes of things we didn't start.

Reply:
Yes, I need this functionality for tests I am adding

Reply:
Then I'll close this. Until your work it wasn't necessary :)